docs/cep/index.md

```diff
@@ -28,6 +28,90 @@ You can process streams to perform the following actions with your data:
 - Extract, transform, and load (ETL) collections, tailing files, and scraping HTTP endpoints.
 - Integrating stream data and trigger actions based on the data. This can be a single service request or a complex enterprise integration flow.
 
+## Stream Worker Explanation
```

Comment:
At some point, we might need to put this in a separate topic. Not today, though.